components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/common/IdentityConnectorConfig.java

```diff
@@ -34,4 +38,9 @@
 
         return null;
     }
+
+    //meta method
+    default Map<String, Property> getMetaData() {
+        return null;
```

Comment:
Formatting issue. There should be a new line after the method signature.

Reply:
Fixed in commit 8032899333352cf8a310623b49856dfb99ec9767

Reply:
Shall we return empty map instead of null